### PR TITLE
Search provider fixes

### PR DESCRIPTION
**Pull request by fushinari**
- [x] Language name crash fix.
- [x] Fix flatpak exporting of search provider ini file
- [x] Fix SP crashing when translating

### CODE REVIEW

File: data/resources/preferences.ui [hunk 1]
```diff
@@ -26,13 +26,22 @@
...
                 <property name="subtitle" translatable="yes">Warning: Your IP address may get banned for API abuse.</property>
-                <property name="activatable_widget">live_translation</property>
+                <property name="show-enable-switch">True</property>
+                <property name="enable-expansion">False</property>
+                <property name="expanded">True</property>
                 <child>
-                  <object class="GtkSwitch" id="live_translation">
-                    <property name="valign">center</property>
+                  <object class="AdwActionRow" id="search_provider">
+                    <property name="title" translatable="yes">Show Translations in Search Provider</property>
```
Comment (rafaelmardojai) [inline data/resources/preferences.ui:37]: This string isn't final. "Search Provider" isn't a user-facing term.

Probably we can discuss this further when checking all the 2.0 new strings.

Comment (fushinari) [reply data/resources/preferences.ui:37]: Yeah, sounds good

File: dialect/search_provider/search_provider.in [hunk 2]
```diff
@@ -51,11 +51,12 @@ class TranslateService(dbus.service.Object):
         on start of input
         """
         self.live_enabled = self.is_live_enabled()
+        backend = Settings.get().active_translator
         text = ' '.join(terms)
         if self.live_enabled:
             self.translation(text)
         else:
-            self.translations[text] = _(f'Translate “{text}” with {Settings.get().active_translator.prettyname}')
+            self.translations[text] = _(f'Translate “{text}” with {TRANSLATORS[backend].prettyname}')
```
Comment (rafaelmardojai) [inline dialect/search_provider/search_provider.in:59]: Now that I think, probably we will need `.format()` here. Does f-string work fine with gettext?

Comment (fushinari) [reply dialect/search_provider/search_provider.in:59]: Nice timing, i had just changed this locally

**Review by rafaelmardojai — APPROVED** [review]

**Pull request merged**